fix build failure on cabal-1.14

Building webkit-0.12.3...
Preprocessing library webkit-0.12.3...
dist/build/Graphics/UI/Gtk/WebKit/Signals.h:1:22: fatal error: hswebkit.h: No such file or directory
compilation terminated.
gtk2hsC2hs: Error during preprocessing custom header file
diff --git a/webkit.cabal b/webkit.cabal
index 8531f2b..1741e05 100644
--- a/webkit.cabal
+++ b/webkit.cabal
@@ -83,5 +83,6 @@ Library
         x-Signals-Types: marshal.list
         x-Signals-Import: Graphics.UI.GtkInternals
 		
+        Include-dirs: .
         x-c2hs-Header:  hswebkit.h
         pkgconfig-depends: webkit-1.0 >= 1.1.15
